src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -39,6 +39,10 @@ public function validate($password, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\UserPassword');
         }
 
+        if (null === $password || '' === $password) {
+            return;
```

Comment:
Well this change only standardizes the usage of the validator, since it already ignored empty strings. It is pretty obvious a `null` cannot be a password, so I would simply do a fix on the side of `FOSUB`